scripts/install.tpl.sh

```diff
@@ -41,13 +41,32 @@ function kubectl_wait_rollout() {
     kubectl rollout status --namespace="${namespace}" "${runtime}" --timeout="${timeout}"
 }
 
+function kubectl_wait_for_query() {
+    manifest=$1
+    query=$2
+    timeout=$3
+    start_time=$(date +%s)
+    while true; do
+        val=$(kubectl get "${manifest}" -o jsonpath="${query}" 2>/dev/null || echo "")
+        if [[ -n "${val}" ]]; then
+            echo "${manifest} has ${query}."
+            break
+        fi
+        if [[ $(( $(date +%s) - start_time )) -ge ${timeout} ]]; then
+            echo "Timed out waiting for ${manifest} to have ${query}."
+            exit 1
+        fi
+        sleep 5
```

Comment:
This is a good change assuming it helps things be less flaky with timing in practice, but

what's the 5 magic number? I think 5s?
1) I think this should be a variable with a good name: `poll_interval_in_seconds`?;
2) But then you are delaying the polling for 5s? Could be a fair amount of extra wait for no benefit.

Reply:
I would even bring back the default to 5. 10 feels like an eternity XD

Reply:
Hi @perdasilva done !!!
It was changed for 5s.
+1 I agree